docs/environments/all-envs.md

```diff
@@ -1,16 +1,26 @@
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|-------------------------|--------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [`catch-v0`](https://rradules.github.io/momaland/environments/catch/) <br><img src="https://rradules.github.io/momaland/_images/catch.gif" width="400px">                               | Cooperative             | Continuous / Continuous | `[distance_target, distance_other_drones]` | Agents must corner and catch a target drone while maintaining distance between themselves.                                                                 |
+| [`escort-v0`](https://rradules.github.io/momaland/environments/escort/) <br><img src="https://rradules.github.io/momaland/_images/escort.gif" width="400px">                            | Cooperative                        | Continuous / Continuous | `[distance_target, distance_other_drones]` | Agents must circle around a mobile target drone and escort it to its destination without breaking formation while maintaining distance between themselves. |
+| [`mobeach-v0`](https://rradules.github.io/momaland/environments/mobeach/) <br><img src="https://rradules.github.io/momaland/_images/mobeach.gif" width="400px">                         |                         | Continuous / Discrete   | `[TODO]`                                   | TODO                                                                                                                                                       |
+| [`mocongestion-v0`](https://rradules.github.io/momaland/environments/mocongestion/) <br><img src="https://rradules.github.io/momaland/_images/mocongestion.gif" width="400px">          |                         | Continuous / Discrete   | `[TODO]`                                   | TODO                                                                                                                                                       |
+| [`moitemgathering-v0`](https://rradules.github.io/momaland/environments/moitemgathering/) <br><img src="https://rradules.github.io/momaland/_images/moitemgathering.gif" width="400px"> | Adversarial                        | Discrete / Discrete     | `[TODO]`                                   | TODO                                                                                                                                                       |
```

Comment:
Same, do we have this gif?

Reply:
There is the single agent resource gathering gif from mogym but [it is not the same](https://mo-gymnasium.farama.org/_images/resource-gathering.gif)

Reply:
You did a gif of this back then, didn't you?